.github/workflows/publish-dev.yaml

```diff
@@ -32,6 +32,12 @@ jobs:
       - name: Build
         run: npm run build
 
+      - name: Test
+        run: npm run test
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+
```

Comment:
Do we need this here? We didn't have lint or test here because presumably that stuff was already done in the pr-checks.

Reply:
We need to run code coverage on the main branch so Codecov can generate a coverage diff.

Reply:
ah, OK. If it's not a requirement to the actual publish, it could be in it's own job then, so we don't slow the publish job down, but not a huge deal.